Makefile

```diff
@@ -189,10 +192,13 @@ sgx/pelz_enclave_u.o: sgx/pelz_enclave_u.c
 	@$(CC) $(App_C_Flags) -c $< -o $@
 	@echo "CC   <=  $<"
 
-bin/$(App_Name): $(App_Cpp_Files) sgx/pelz_enclave_u.o
+bin/$(App_Name_Service): $(App_Service_File) $(App_Cpp_Files) sgx/pelz_enclave_u.o
 	@$(CXX) $^ -o $@ $(App_Cpp_Flags) $(App_Include_Paths) -Isgx $(App_C_Flags) $(App_Link_Flags) -Lsgx -lcrypto -lcjson -lpthread
 	@echo "LINK =>  $@"
 
+bin/$(App_Name_Pipe): $(App_Pipe_File) $(App_Cpp_Files) sgx/pelz_enclave_u.o
```

Comment:
With the current functionality we don't need to include the enclave stuff here (or in the linker flags.) I'd vote for removing anything that's not necessary right now, we can always add them back later if we need to.

Reply:
Without enclave, "make" has reference pointer errors.